clopez-applaudo/vue-playground
# Feature/hooks

**Status**
Add pre-commit and pre-push hooks using Husky.
Pre-commit will run:

- eslint using configuration that includes vue3 and prettier rules
- Prettier through the library pretty-quick in order to check and update only staged elements.

Pre-push will run:
- Vitest test suit running once with basic configuration for Vue.

**Evidence**
![image](https://github.com/clopez-app/vue-playground/assets/61519643/be4b393f-b205-442c-bd0b-7cb1f1e7929a)

**How to test**
Make sure to use npm install again if you did it before this merge to install husky in the environment.
Check configuration on the files.
If you need to commit/push code you will be able to test both hooks in action, otherwise please just check on the configuration.
